tests/kuttl/common/assert-sample-deployment.yaml

```diff
@@ -0,0 +1,140 @@
+  creationTimestamp: "2023-02-14T14:50:07Z"
+  finalizers:
+  - Ironic
+  generation: 1
+  name: ironic
+  namespace: openstack
+  resourceVersion: "75736"
+  uid: 17c6557d-8e10-46fc-8ce7-9ba67bab2b1b
+spec:
+  customServiceConfig: |
+    [DEFAULT]
+    debug = true
+  databaseInstance: openstack
+  databaseUser: ironic
+  ironicAPI:
+    containerImage: quay.io/tripleozedcentos9/openstack-ironic-api:current-tripleo
+    customServiceConfig: '# add your customization here'
+    databaseUser: ironic
+    passwordSelectors:
+      database: IronicDatabasePassword
+      service: IronicPassword
+    replicas: 1
+    rpcTransport: json-rpc
+    serviceUser: ironic
+    standalone: false
+  ironicConductor:
+    containerImage: quay.io/tripleozedcentos9/openstack-ironic-conductor:current-tripleo
+    customServiceConfig: '# add your customization here'
+    databaseUser: ironic
+    passwordSelectors:
+      database: IronicDatabasePassword
+      service: IronicPassword
+    pxeContainerImage: quay.io/tripleozedcentos9/openstack-ironic-pxe:current-tripleo
+    replicas: 1
+    rpcTransport: json-rpc
+    serviceUser: ironic
+    standalone: false
+    storageClass: local-storage
+    storageRequest: 10G
+  ironicInspector:
+    containerImage: quay.io/tripleozedcentos9/openstack-ironic-inspector:current-tripleo
+    customServiceConfig: '# add your customization here'
+    databaseUser: ironic_inspector
+    passwordSelectors:
+      database: IronicInspectorDatabasePassword
+      service: IronicInspectorPassword
+    preserveJobs: true
+    pxeContainerImage: quay.io/tripleozedcentos9/openstack-ironic-pxe:current-tripleo
+    rabbitMqClusterName: rabbitmq
+    replicas: 1
+    rpcTransport: json-rpc
+    serviceUser: ironic-inspector
+    standalone: false
+  passwordSelectors:
+    database: IronicDatabasePassword
+    service: IronicPassword
+  preserveJobs: true
+  rabbitMqClusterName: rabbitmq
+  rpcTransport: json-rpc
+  secret: osp-secret
+  serviceUser: ironic
+  standalone: false
+status:
+  apiEndpoints:
+    ironic:
+      admin: http://ironic-admin-openstack.apps-crc.testing
```

Comment:
The urls for the endpoints will be slightly different for different environament (CRC vs SNO), a regex-based approach will be needed. See https://github.com/openstack-k8s-operators/keystone-operator/pull/151

Reply:
We will address this in a future commit.  Thank you for pointing this out!